development/image-detector/README.md

```diff
@@ -0,0 +1,27 @@
+# image-detector
+
+Image Detector is a tool used to update security scanner config with list of images that lives in our prow cluster. To achive that it receives paths to files used to deploy prow or it's components.
```

Comment:
I've changed `our` to `your`, as the latter form is used as a general possessive adjective. You didn't use `our` meaning `Neighbors'`, right?

Reply:
Not `Neighbors`, but 'Kyma'.

WOuld be better to use something like: `list of images that lives in prow cluster`?